examples/downstream-v0.sh

```diff
@@ -1,16 +1,44 @@
 #!/bin/bash
-#
-# This script will run a local OLMv0 downstream operation, to allow someone to
-# debug a failure.
-#
-# Assumptions:
-# * The tooling directory is located in ${HOME}/git/openshift/operator-framework-tooling
-# * The v0 executable has been built in there via `make`
-#
-# Call with your github user name, if it's not the same as your current user name.
-#
-
-GITHUB_USER=${1:-${USER}}
+set -euo pipefail
+
+usage() {
+    echo "This script will run a local OLMv0 downstream operation, to allow someone to"
+    echo "debug a failure."
+    echo ""
+    echo "Usage: $0 [-u github-username] [-a \"-extra-sync args\"] [-d /path/to/the/sync/dir]" 1>&2;
```

Comment:
A bit concerned that this `1>&2` (stdout to stderr) might be done out of order... But maybe that can't happen. But still wondering why?

Reply:
Good catch! I think this is a copy-paste error. I'll get rid of it.